package.json

```diff
@@ -22,6 +22,9 @@
     "start": "craco start",
     "build": "craco build",
     "test": "craco test",
+    "cypress:open": "cypress open",
+    "cypress:ci": "cypress run --config video=false",
```

Comment:
@tanvirchahal I think this is the command you need to run in CI so it passes

Reply:
it would be great if we could rename this command to `e2e:ci`

Reply:
@anuveyatsu Updated the commands to be 
`"e2e:interactive": "cypress open",
    "e2e:ci": "cypress run --config video=false",
    "e2e": "cypress run"`